osu.Game/Configuration/OsuConfigManager.cs

```diff
@@ -102,6 +102,8 @@ protected override void InitialiseDefaults()
 
             SetDefault(OsuSetting.MenuParallax, true);
 
+            SetDefault(OsuSetting.Prefer24HourTime, false);
```

Comment:
Hmm, thinking about this a second time, I'm going to propose that this is set to `true` by default (I think it's a more globally accepted default) and the per-culture check is removed.

Reasoning is that many countries default to showing AM/PM but a majority would likely prefer 24 hour time display. I may be wrong though.

Will wait for further opinions on the matter.

Reply:
Might as well fetch the current system culture and set the default value based on it (one way would be checking if [`LongTimePattern`](https://docs.microsoft.com/en-us/dotnet/api/system.globalization.datetimeformatinfo.longtimepattern?view=net-6.0#system-globalization-datetimeformatinfo-longtimepattern) contains `tt`, the AM/PM specifier).

Reply:
That's already done in this PR, in a slightly different location. But my point remains that that may not be the best choice for the user (I know that's the case for myself). My proposal is to always start with 24 hour time preferred.

Reply:
Oh, well I find it weird that the setting can only be effective based on the current culture info (unless it's done that way for languages which don't support it). 

But I personally think it would be best for the default value of this setting to be based on the user's system culture configuration (which I hope `InstalledUICulture` covers).

Otherwise, enforcing to 24-hours system sounds at least better.

Reply:
The current implementation with `CurrentCulture` works properly for both English (US) and French (France) languages, but it looks like it incorrectly displays 12-hour time if the Windows display language is English but the regional format is French, and that this behavior is fixed by using `InstalledUICulture`, so if we decide to use some sort of automatic clock formatting I'll definitely update that.

Defaulting to 24-hour and letting the user change it would be perfectly fine I think (I've had to do it before with other applications and it's not exactly a tedious process) but if we do want to keep detection based off the culture it might be a good idea to have a dropdown with Automatic/12-hour/24-hour instead of a single toggle for 24-hour. This also has the added benefit of giving users with a culture that defaults to 24-hour the option of switching to 12-hour (currently, there is no way for these users to use the 12-hour clock).

Reply:
Automatic would not be required with the proposed behaviour.

Reply:
As is tradition, I would personally much rather not have this setting at all, but if it's here, then false seems like a better default.

Reply:
Okay, let's go with that.